packages/typescript/types.ts

```diff
@@ -528,6 +530,8 @@ export interface PrettyResponse {
 
 export type DictStrStr = { [key: string]: string };
 
+export type DictStrList = { [key: string]: string[] };
```

Comment:
i think there is no concept of `Dict[str, List[Optional[str]]]`
in ts it would just be `Dict[str, List[str]]`